kolibri/plugins/facility/assets/src/views/FacilityConfigPage/CreateManagementPinModal.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+
+  <KModal
+    :title="$tr('title')"
+    :submitText="$tr('save')"
+    :cancelText="$tr('cancel')"
+    @submit="$emit('submit')"
+    @cancel="$emit('cancel')"
+  >
+    <div>
+      <p>{{ $tr('newToSync') }}</p>
+      <p>{{ $tr('enterPin') }}</p>
+
+      <KTextbox
+        v-model="name"
+        type="number"
```

Comment:
A **non-blocking** comment: 

I can totally understand why you decided to add type 'number' here, since, it's a number. However, number text fields do have the increment/decrement arrows.

<img width="427" alt="Screenshot 2023-01-20 at 11 11 10 AM" src="https://user-images.githubusercontent.com/17235236/213747984-7f44d770-c312-4a09-84ef-6849fd8f9626.png">

I wonder if having this alongside the need for a 4 digit input might be confusing? Part of me thinks yes, and part of me thinks "people know what a PIN is, nobody is going to click the arrow 1000 times to generate a 4 digit PIN".... 

Curious if you, @AllanOXDi, or @akolson or @rtibbles have any thoughts.

Reply:
On the flip side, specifying `number` means that on devices with on screen keyboards, it will display an on screen keyboard with just numeric input, which is a distinct usability win.

That said, it looks like we can get the same result by using an input type text or password and adding the `inputmode` attribute set to `numeric` https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode

So, I'd suggest using a text input, with the `inputmode` set to `numeric` and then setting the `pattern` attribute to do the number validation: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#pattern

Can use a simple regex like `[0-9]{4}` to ensure the 4 digit PIN?

Reply:
Hi @marcellamaki @rtibbles, Thanks for pointing this out and thanks for the suggestions.